src/main/java/org/broadinstitute/hellbender/engine/TwoPassReadWalker.java

```diff
@@ -0,0 +1,96 @@
+package org.broadinstitute.hellbender.engine;
```

Comment:
I'm not sure if we want to support this as part of the engine or just make it a package protected class and move it to rnaseq.  If we want to keep it in engine then we should add integration tests for it with an example walker.